launch.go

```diff
@@ -308,9 +308,17 @@ func SetExternalMeta(api screwdriver.API, pipelineID, parentBuildID int, mergedM
 			if join {
 				resultMeta = deepMergeJSON(parentBuild.Meta, resultMeta)
 			}
-			externalMetaKey := "sd." + strconv.Itoa(parentJob.PipelineID) + "." + parentJob.Name
-			// delete stored external meta key
-			delete(resultMeta, externalMetaKey)
+
+			// set stored external meta key to "null"
+			pIDString := strconv.Itoa(parentJob.PipelineID)
+			pjn := parentJob.Name
+			if sdMeta, ok := resultMeta["sd"]; ok {
+				if externalPipelineMeta, ok := sdMeta.(map[string]interface{})[pIDString]; ok {
+					if _, ok := externalPipelineMeta.(map[string]interface{})[pjn]; ok {
+						externalPipelineMeta.(map[string]interface{})[pjn] = "null"
```

Comment:
I think this can be `delete(externalPipelineMeta, pjn)` to remove the job key from `externalPipelineMeta`